internal/common/game.go

```diff
@@ -2,17 +2,18 @@ package common
 
 import "github.com/SOMAS2020/SOMAS2020/internal/common/shared"
 
-// DefaultResources is the default number of resources at the start of the game
-const DefaultResources = 100
-
 // ClientInfo contains the client struct as well as the client's attributes
 type ClientInfo struct {
-	Client Client
-
 	// Resources contains the amount of resources owned by the client.
-	Resources uint
+	// Made an integer so an island can "owe" resources.
+	Resources int
 
-	Alive bool
+	// Normal Condition: Alive = true, Critical = false, CriticalConsecutiveTurnsLeft = config.MaxCriticalConsecutiveTurnsLeft

```

Comment:
This should be an enum because having separate bools allows for invalid states (e.g. dead and non-critical)

Reply:
Yep, will change -- wanted to do it but the critical days left was a bit of an ass to do, but it should make sense